<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloorPlanDialog</class>
 <widget class="QDialog" name="FloorPlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FloorPlan Reconstruction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Insistent Region Detection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_KNN">
          <property name="text">
           <string>&amp;Nearest Neighbours Number</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_knn</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_knn"/>
        </item>
        <item>
         <widget class="QLabel" name="label_Threshold">
          <property name="text">
           <string>&amp;Threshold</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBox_threshold</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_threshold"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dispInconsist">
          <property name="text">
           <string>Display Inconsist Region</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_insistRegionDetection">
          <property name="text">
           <string>Detecting</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox_gettingSlices">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Getting Slices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_stepValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label_setStepValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Step Value</string>
            </property>
            <property name="buddy">
             <cstring>doubleSpinBox_stepValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_setLayerNumber">
            <property name="text">
             <string>Layer Numbers</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_displayLayerNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dispSices">
          <property name="text">
           <string>Diplay Slices</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>331</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_divide_slices">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Dividing</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_mainDirecionNum">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Main Direction Numbers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_dispMainDirectionNumbers">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_minmum_angle">
          <property name="text">
           <string>Minmum Angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_minAngle"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_mainDireciongs">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Computint Main Directions</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox_gettingFloorPlan">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Get Floor Plan Each Layer</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_setEndingLayer">
          <property name="text">
           <string>Ending Layer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_Slayer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_setStartingLayer">
          <property name="text">
           <string>Sarting Layer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_Elayer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="Label_setLineWidth">
          <property name="text">
           <string>&amp;Line Width</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBox_lineWidth</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_lineWidth"/>
        </item>
        <item>
         <widget class="QLabel" name="Label_setStepValue_3">
          <property name="text">
           <string>&amp;Curve Margin</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBox_curveMargin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_curveMargin"/>
        </item>
        <item>
         <widget class="QLabel" name="Label_setMRFValue">
          <property name="text">
           <string>&amp;MRF Optimization Params</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBox_MRFLambda</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_MRFLambda"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Letter Margin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_letterMargin"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_dispProcess">
          <property name="text">
           <string>Display Reconstruction Process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dispModel">
          <property name="text">
           <string>Display Final Model</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_floorplanRec">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox_modelingCeiling">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Modeling Ceiling</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_edit_mode">
          <property name="text">
           <string>Edit Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addCeilingPlane">
          <property name="text">
           <string>Add Ceiling plane</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_acceptFloorPlygon">
            <property name="text">
             <string>Accept</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cancelFloorPolygon">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>71</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_adjustment_mode">
          <property name="text">
           <string>Adjustment Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QPushButton" name="pushButton_selecPointsOnEditPlane">
            <property name="text">
             <string>Select Points</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_bundleVertices">
            <property name="text">
             <string>Bundle Planes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pushButton_accetpAdjusment">
            <property name="text">
             <string>Accept</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cancelAdjustment">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QPushButton" name="pushButton_birdView">
            <property name="text">
             <string>Bird View</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_lateralView">
            <property name="text">
             <string>Lateral View</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_DTsOnEditPlane">
          <property name="text">
           <string>Delaunay Triangulation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox_IGM">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Image Guided Modeling</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_PolygonPlanes">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Get Ploygon Planes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_beginGTSDetection_">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>GTS Detection</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_backprojection">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Back Projection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>154</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Window Depth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_windowDepth"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_addWindowPlanes">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add Window Planes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="pushButton_accpectWindowPlanes">
            <property name="text">
             <string>Accept </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cacelWindowPlanes">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>GroupBox_gettingSlices</zorder>
  <zorder>GroupBox_gettingFloorPlan</zorder>
  <zorder>GroupBox_modelingCeiling</zorder>
  <zorder>GroupBox_IGM</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_insistRegionDetection</tabstop>
  <tabstop>pushButton_divide_slices</tabstop>
  <tabstop>pushButton_mainDireciongs</tabstop>
  <tabstop>horizontalSlider_Slayer</tabstop>
  <tabstop>horizontalSlider_Elayer</tabstop>
  <tabstop>pushButton_floorplanRec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
